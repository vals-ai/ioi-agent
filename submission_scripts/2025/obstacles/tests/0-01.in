version https://git-lfs.github.com/spec/v1
oid sha256:3c191180690762efd29d9a74c11a1ac37c2b28ec8e9b8940f4d8f3c15966123f
size 69
