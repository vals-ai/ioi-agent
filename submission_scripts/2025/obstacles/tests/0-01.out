version https://git-lfs.github.com/spec/v1
oid sha256:672978fc05ffb1cb8cbf379550201ecf244dae6378dc7cf4793b67dd36c98a87
size 40
